tests/regression/test_nullspace.py

```diff
@@ -341,8 +351,6 @@ def test_near_nullspace_mixed():
             'pc_type': 'python',
             'pc_python_type': 'firedrake.AssembledPC',
             'assembled_pc_type': 'gamg',
-            'assembled_pc_gamg_aggressive_square_graph': None,  # See https://gitlab.com/petsc/petsc/-/commit/d529f056d75cd16b380adc3e50ccb741df53d0de
-            'assembled_pc_gamg_mis_k_minimum_degree_ordering': True,  # See https://gitlab.com/petsc/petsc/-/commit/d529f056d75cd16b380adc3e50ccb741df53d0de
```

Comment:
Is removing of these options still correct, @stephankramer I know you've had a bunch of back and worth with Mark around gamg changes recently.